{"name": "test_submitbug", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <TestCase.test_ceshi.test_submit_bug.TestSubmitBug object at 0x105b39748>\ndrivers = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"97a0e099a0a23b42234a58619f14bc62\")>\ntitle = '第二个bug', content = 'bug内容'\n\n    def test_submitbug(self,drivers,title='第二个bug',content='bug内容'):\n        self.bug = BugPage(drivers)\n>       self.bug.submit_bug(title,content)\n\nTestCase/test_ceshi/test_submit_bug.py:21: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nPage/TestPage/bugpage.py:36: in submit_bug\n    self.click(self.submitbug_ele,doc=self.Doc)\nCommon/base.py:133: in click\n    ele = self.find(locator,doc)\nCommon/base.py:90: in find\n    ele = WebDriverWait(self.driver,self.time,self.timeout).until(lambda x:x.find_element(*locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"97a0e099a0a23b42234a58619f14bc62\")>\nmethod = <function BasePage.find.<locals>.<lambda> at 0x10de1ea60>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/selenium/webdriver/support/wait.py:80: TimeoutException"}, "attachments": [{"name": "测试模块--提交Bug功能--", "source": "cefde94d-4b08-4aac-8178-19f8988521a7-attachment.png", "type": "image/png"}, {"name": "log", "source": "7c552b50-c20b-47a2-8165-e37adb220556-attachment.txt", "type": "text/plain"}], "start": 1598632194816, "stop": 1598632207773, "uuid": "73346b9b-4ccb-45ad-93af-2085a90a1cb2", "historyId": "96d448f71e9f3cc47b09982c5bb23188", "fullName": "TestCase.test_ceshi.test_submit_bug.TestSubmitBug#test_submitbug", "labels": [{"name": "parentSuite", "value": "TestCase.test_ceshi"}, {"name": "suite", "value": "test_submit_bug"}, {"name": "subSuite", "value": "TestSubmitBug"}, {"name": "host", "value": "zhangbijiangdeMacBook-Pro.local"}, {"name": "thread", "value": "14835-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.test_ceshi.test_submit_bug"}]}